### PR TITLE
Add Sequential api

### PR DESCRIPTION
**Pull request by songyouwei**
support `Sequential` container

### CODE REVIEW

File: python/paddle/fluid/dygraph/container.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                model1[0]  # access fc1 layer
+                res1 = model1(data)  # sequential execution
+
+                # create Sequential with name Layer pairs
+                model2 = fluid.dygraph.Sequential('model2',
+                    ('l1', fluid.FC('l1', 2)),
+                    ('l2', fluid.FC('l2', 3))
+                )
+                model2['l1']  # access l1 layer
+                model2.add_sublayer('l3', fluid.FC('l3', 3))  # add sublayer
```
Comment (JiabinYang) [inline python/paddle/fluid/dygraph/container.py:52]: add some comments to show where `l3` will be added

File: python/paddle/fluid/tests/unittests/test_imperative_container.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
```
Comment (phlrain) [inline python/paddle/fluid/tests/unittests/test_imperative_container.py]: test的名字 是不是可以叫 test_imperative_container_sequential.py, 后面可能会增加别的container

**Review by JiabinYang — COMMENTED** [review]
just one comment, and better let @phlrain review this

File: python/paddle/fluid/tests/unittests/test_imperative_container.py (hunk 2 above)
Comment (songyouwei) [reply python/paddle/fluid/tests/unittests/test_imperative_container.py]: done

File: python/paddle/fluid/dygraph/container.py (hunk 1 above)
Comment (songyouwei) [reply python/paddle/fluid/dygraph/container.py:52]: done

**Review by phlrain — DISMISSED** [review]

**Review by JiabinYang — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]
LG API

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**